src/docs/release/breaking-changes/route-transition-record-and-transition-delegate.md

```diff
@@ -0,0 +1,139 @@
+---
+title: The Route Transition Record and Transition Delegate Updates
+description: Changes to the rule on how transition delegate resolve route transition.
+---
+
+## Summary
+
+Added a new boolean getter `isWaitingForExitingDecision` to the route
+transition record and renamed the getter `isEntering` to
+`isWaitingForEnteringDecision`. During the resolve method in transition
```

Comment:
codefont `resolve`

the resolve method in transition delegate => the `resolve` method in the transition delegate. (??)